statistics/sample_test.go

```diff
@@ -107,7 +107,7 @@ func (s *testSampleSuite) TestMergeSampleCollector(c *C) {
 	c.Assert(len(collectors), Equals, 2)
 	collectors[0].MergeSampleCollector(collectors[1])
 	c.Assert(collectors[0].Sketch.NDV(), Equals, int64(9280))
-	c.Assert(len(collectors[0].Samples), Equals, 10000)
+	c.Assert(len(collectors[0].Samples), Equals, 1000)
```

Comment:
The change of the test doesn't look like covering this bug.

Reply:
It covers, origin code will make the `Count` at line 112 not equal to `19000`